src/OpenRiaServices.Tools/Test/CodeGenDomainServices.cs

```diff
@@ -166,6 +166,30 @@ public void TestClientCodegen_EFDbCtxDomainServices_FullTypes()
                 typeof(TestDomainServices.DbCtx.Northwind), sharedFiles, true));
         }
 
+        [DeploymentItem(@"Baselines\Default\Scenarios", "CG_Scenarios_EFCoreContext")]
+        [DeploymentItem(@"ProjectPath.txt", "CG_Scenarios_EFCoreContext")]
+        [TestMethod]
+        public void TestClientCodegen_EFCoreDomainServices()
+        {
+            string[] sharedFiles = Array.Empty<string>();
+
+            // Default codegen
+            TestHelper.ValidateCodeGen(new TestHelper.CodeGenValidationOptions(@"Default\Scenarios", "CG_Scenarios_EFCoreContext", "EFCoreContextScenarios.g",
+                typeof(TestDomainServices.EFCore.Northwind), sharedFiles, false));
+        }
+
+        [DeploymentItem(@"Baselines\FullTypeNames\Scenarios", "CG_Scenarios_EFDbContext_FullTypes")]
+        [DeploymentItem(@"ProjectPath.txt", "CG_Scenarios_EFDbContext_FullTypes")]
```

Comment:
you should be able to reuse same folder name (second parameter) as above to not get to manu extra testfolders